assets/sass/404.scss

```diff
@@ -17,9 +18,11 @@ $position-left: 27vw;
 
 // The content is put inside of #clouds-and-beams.
 #clouds-and-beams {
-    color: #fcf7e7;
-    font-family: Oxygen;
+    color: map-get($gray, 100);
     line-height: 1.1;
+    font-family: "Open Sans", Arial, Helvetica, sans-serif;
```

Comment:
Any reason why we define a custom font family here and not the existing font stack we use in `pulumi.scss`? (Is this any different from what we serve normally?)

Reply:
Yep -- it's only because we define the font stack for the rest of the site [in tailwind.config.css](https://github.com/pulumi/docs/blob/dd6043bad71491342c7309118b8d835786b159d2/assets/config/tailwind.config.js#L80-L83), and this page is currently divorced from the Tailwind stuff, so it doesn't have access to any of those classes. 

I started a WIP branch to refactor this page into a layout built with Tailwind and the rest of our new CSS, but it ended up a little bigger that I wanted to bite off today, so I made the necessary changes for the issue, and figured we could come back later to do the rest.